<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>convertWindow</class>
 <widget class="QMainWindow" name="convertWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>139</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FreeBox Converter 1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="file_layout">
      <item>
       <widget class="QLineEdit" name="input_file"/>
      </item>
      <item>
       <widget class="QLineEdit" name="output_file"/>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="input_choose">
        <property name="text">
         <string>Input File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="output_choose">
        <property name="text">
         <string>Output File</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="convert_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Convert</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>input_choose</tabstop>
  <tabstop>output_choose</tabstop>
  <tabstop>convert_button</tabstop>
  <tabstop>input_file</tabstop>
  <tabstop>output_file</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
